chains/basesepolia/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-sepolia.basescan.org/api
+    family: etherscan
+    name: BaseScan
+    url: https://sepolia.basescan.org/
```

Comment:
Nit: avoid trailing slashes in these URLs please :)

Reply:
updated